sycl/cmake/modules/AddSYCLUnitTest.cmake

```diff
@@ -157,8 +157,8 @@ endfunction()
 #
 # Will compile the list of files together to create two builds, with and without
 # the SYCL preview features enabled.
-# Produces two binaries, named `basename(test_name_prefix_non_preview)` and `basename(test_name_prefix_preview)`
+# Produces two binaries, named `basename(test_name_prefix_Non_Preview_Tests)` and `basename(test_name_prefix_Preview_Tests)`
```

Comment:
NIT: Can we please add a comment here about the suffix assumption we make? So that in future, if we change the names of these tests, we'd have to accordingly update `sycl/test/Unit/lit.cfg.py`

Reply:
Added in https://github.com/intel/llvm/pull/20534/commits/02ae68a0bd367cd01df2aff386d3abc9bc00a366, can you review the wording?